src/index.ts

```diff
@@ -1640,23 +1641,33 @@ export class KycDao extends ApiBase {
           this.near.contractName,
           {
             viewMethods: [],
-            changeMethods: ['mint'],
+            changeMethods: ['mint_with_code', 'get_required_mint_cost_for_code'],
```

Comment:
Isn't `get_required_mint_cost_for_code` supposed to be a view method? Maybe I have a wrong interpretation of these parameters :D